src/test/java/dev/openfeature/sdk/integration/StepDefinitions.java

```diff
@@ -39,15 +53,34 @@ public class StepDefinitions {
     private String notFoundFlagKey;
     private String notFoundDefaultValue;
     private FlagEvaluationDetails<String> notFoundDetails;
-    private String typeErrorFlagKey;
     private int typeErrorDefaultValue;
     private FlagEvaluationDetails<Integer> typeErrorDetails;
 
-    @BeforeAll()
-    @Given("an openfeature client is registered with cache disabled")
-    public static void setup() {
-        // TODO: when the FlagdProvider is updated to support caching, we might need to disable it here for this test to work as expected.
-        FlagdProvider provider = new FlagdProvider();
+    @BeforeAll
+    public static void init() {
+        ObjectMapper mapper = new ObjectMapper();
+        try {
+            flagConfig = mapper.readTree(new File(testingFlagsPath));
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
let's use logger with `@Slf4j` annotation  for all such places

Reply:
thanks, done